journey-map/public/index.mjs

```diff
@@ -187,7 +226,13 @@ export const render = (journeyMap, nestedJourneys, formData = new FormData()) =>
     }
     expandParents(journeyMapCopy);
 
-    const { transitionsMermaid, states } = renderTransitions(journeyMapCopy, formData);
+    let { transitionsMermaid, states } = renderTransitions(journeyMapCopy, formData);
+
+    if (formData.has('onlyOrphanStates')) {
+        states = calcOrphanStates(journeyMapCopy)
+        transitionsMermaid = '';
+    }
```

Comment:
I think the ternary is a little less clear in this scenario but it does solve something I was hoping you'd comment on.
The way I have it currently I'm mutating `transitionsMermaid` and `states` after the call to `renderTransitions()`. That force me to use `let` instead of `const`.
I was trying for a little while to do something like:
```
    let states;
    let transitionsMermaid;
    if (formData.has('onlyOrphanStates')) {
        states = calcOrphanStates(journeyMapCopy)
        transitionsMermaid = '';
    } else {
        { transitionsMermaid, states } = renderTransitions(journeyMapCopy, formData);
    }
```
But the else block doesn't work - variable might not have been initialized. I think the brace return thing is throwing me a little. How would you handle something like this? I guess use the ternary?

Reply:
This is a case I had to look up, actually! Turns out you have to use an extra pair of brackets...
```
} else {
    ({ transitionsMermaid, states } = renderTransitions(journeyMapCopy, formData));
}
```
I prefer the ternary tbh - it makes it clear that both are being set in both branches

Reply:
Ah yeah, of course, extra brackets! 🫤

But overall the ternary does seem like the best bet

Reply:
(I suppose making the `calcOrphanStates` method also return a blank `transitionsMermaid` would also be a neat, but not very satisfying, solution)